README.md

```diff
@@ -215,15 +218,21 @@ Inspired by [awesome-python](https://github.com/vinta/awesome-python).
 
 *Tools for managing model lifecycle (tracking experiments, parameters and metrics).*
 
+* [AIF360](https://github.com/Trusted-AI/AIF360) - A comprehensive set of fairness metrics for datasets and machine learning models.
 * [Aim](https://github.com/aimhubio/aim) - A super-easy way to record, search and compare 1000s of ML training runs.
 * [Comet](https://github.com/comet-ml) - Track your datasets, code changes, experimentation history, and models.
+* [Fairlearn](https://github.com/fairlearn/fairlearn) - A Python package to assess and improve fairness of machine learning models.
```

Comment:
What do you think about creating a new section called "Model Fairness" and including `Fairlearn` and `AIF360` there?

Reply:
I suggest to add new section called "Model Fairness and Privacy" and including `Fairlearn`, `AIF360`, `TensorFlow Privacy `and `Opacus` there, what do you think?